gems/pending/spec/postgres_ha_admin/failover_databases_spec.rb

```diff
@@ -1,32 +1,48 @@
-describe FailoverDatabases do
+require 'postgres_ha_admin/failover_databases'
+require 'postgres_ha_admin/postgres_ha_admin'
+require 'spec/support/test_env_helper'
+require 'pg'
+
+describe PostgresHaAdmin::FailoverDatabases do
+  subject do
+    described_class.new(TestEnvHelper::SPEC_DIR, TestEnvHelper::SPEC_DIR, :dbname => 'travis', :user => 'travis')
+    # described_class.new(TestEnvHelper::SPEC_DIR, TestEnvHelper::SPEC_DIR, :dbname => 'vmdb_test')
+  end
+
   before do
-    connection = ActiveRecord::Base.connection
-    connection.execute("CREATE SCHEMA repmgr_miq")
+    drop_schema
+    connection = pg_connection
+    connection.exec("CREATE SCHEMA repmgr_miq")
 
-    connection.execute(<<-SQL)
+    connection.exec(<<-SQL)
       CREATE TABLE repmgr_miq.repl_nodes (
       type text NOT NULL,
       conninfo text NOT NULL,
       active boolean DEFAULT true NOT NULL)
     SQL
 
-    connection.execute(<<-SQL)
+    connection.exec(<<-SQL)
       INSERT INTO
         repmgr_miq.repl_nodes(type, conninfo, active)
       VALUES
         ('master', 'host=1.1.1.1 user=root dbname=vmdb_test', 'true'),
         ('standby', 'host=2.2.2.2 user=root dbname=vmdb_test', 'true'),
         ('standby', 'host=3.3.3.3 user=root dbname=vmdb_test', 'false')
     SQL
+    connection.finish
   end
 
   after do
     remove_file
   end
 
+  after(:all) do
```

Comment:
Instead of doing this you may be able to roll your own "transactional fixtures" with something like the following:
```ruby
let(:connection) { PG::Connection.open(:dbname => "postgres") }

around(:each) do |example|
  begin
    connection.exec("BEGIN")
    example.run
  ensure
    connection.exec("ROLLBACK")
  end
end
```'

That way you don't have to have a `pg_connection` method and you don't have to worry about dropping the schema at the end or at the start.